rewrite-groovy/src/groovy2Test/java/org/openrewrite/groovy/GroovyParserTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.groovy;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.tree.ParseError;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class GroovyParserTest {
+
+    @Test
+    void souldNotTreatDivisionAsDelimiter() {
```

Comment:
Just curious, but was there a reason for this not to be a `rewriteRun(..)`? It should verify that everything parses correctly into the `G` type as well as print idempotency.

Reply:
Nope, changed to use `rewriteRun`! Also added a test case to test a mix of `SLASHY_STRING` and division.